test/e2e-go/cli/algoh/expect/algohExpectCommon.exp

```diff
@@ -0,0 +1,88 @@
+# Algoh Expect Utility Package
+namespace eval ::Algoh {
+
+  # Export Procedures
+  namespace export Info
+  namespace export EstablishNetwork
+
+
+  # My Variables
+   set version 1.0
+   set description "Algodh Expect Package"
+
+  # Variable for the path of the script
+   variable home [file join [pwd] [file dirname [info script]]]
+}
+
+# Definition of the procedure MyProcedure
+proc ::Algoh::Info {} {
+   puts Algoh::description
+}
+
+package provide Algoh $Algoh::version
+package require Tcl 8.0
+
+
+# Start the network
+proc ::Algoh::EstablishNetwork { NETWORK_NAME NETWORK_TEMPLATE TEST_ROOT_DIR } {
```

Comment:
Could you please break this into it's components ?
i.e. CreateNetwork and StartNetwork ?

Reply:
yes will do